frontend/scripts/react-components/chart/chart.component.jsx

```diff
@@ -128,28 +115,8 @@ class Chart extends PureComponent {
             {cartesianGrid && (
               <CartesianGrid strokeDasharray="4 4" stroke="#d6d6d9" {...cartesianGrid} />
             )}
-
-            {xAxis && <XAxis dataKey={xKey || ''} tick={{ fontSize: 12 }} {...xAxis} />}
-
-            {yAxis && (
-              <YAxis
-                axisLine={false}
-                tickSize={-50}
-                mirror
-                tickMargin={0}
-                tickLine={false}
-                tick={
-                  <ChartTick
-                    dataMax={maxYValue}
-                    unit={unit || ''}
-                    unitFormat={unitFormat || (value => value)}
-                    fill="#555555"
-                  />
-                }
-                {...yAxis}
-              />
-            )}
-
+            {CustomXAxis({ config, data })}
```

Comment:
Why not mount it as a component?

Reply:
I tried but the ComposedChart in recharts only accept some specific components. If you include something else it doesn't render it without a warning